### PR TITLE
Add dragTargetWidth option

### PR DESCRIPTION
**Pull request by nekonenene**
## Proposed changes

The sidebar is draggable for mobile, but when I actually operate it on my smartphone, it is a little difficult to start dragging it. Therefore, I added an option it would be possible to adjust the width of the druggable area.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue).
- [x] New feature (non-breaking change which adds functionality).
- [ ] Breaking change (fix or feature that would cause existing functionality to change).

## Checklist:

- [x] I have read the **[CONTRIBUTING document](https://github.com/materializecss/materialize/blob/master/CONTRIBUTING.md)**.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: js/sidenav.js [hunk 1]
```diff
@@ -394,6 +396,7 @@
     _createDragTarget() {
       let dragTarget = document.createElement('div');
       dragTarget.classList.add('drag-target');
+      dragTarget.style.width = this.options.dragTargetWidth;
```
Comment (Smankusors) [inline js/sidenav.js:399]: hmm if we set the width via `style` attribute, will this line from SCSS necessary?

https://github.com/materializecss/materialize/blob/7bf14e073402dcc620235758e248dd83acfe0844/sass/components/_sidenav.scss#L152

Comment (nekonenene) [reply js/sidenav.js:399]: Certainly, there are no problem to delete the line. Thanks!

**Review by roiLeo — APPROVED** [review]

**Comment by Smankusors** [comment]
just a reminder, setting this option too big will be a problem with other element interactions, such as button clicks.

otherwise, this is really good, I once encountered this but never bothered haha. Thanks 😅

**Review by DanielRuf — APPROVED** [review]

**Comment by nekonenene** [comment]
@Smankusors 
Yes, that's right! We can think more about better values of z-index.

**Pull request merged**